spec/vcr_configurer.rb

```diff
@@ -34,8 +34,17 @@ def self.filter_header(vcr_config, part, header_name)
         client_path = client_request.parsed_uri.path
         recorded_path = recorded_request.parsed_uri.path
         uploads_regex = %r{(\/gdc\/uploads\/)[^\/]+(.*)}
+        project_regex = %r{(\/gdc\/projects\/)[^\/]+(.*)}
+        process_regex = %r{(\/gdc\/projects\/)[^\/]+/dataload/processes/[^\/]+}
+        schedule_regex = %r{(\/gdc\/projects\/)[^\/]+/schedules/[^\/]+}
         if client_path.match(uploads_regex) && recorded_path.match(uploads_regex)
           client_path.gsub(uploads_regex, '\1UPLOADS_TMP\2') == recorded_path.gsub(uploads_regex, '\1UPLOADS_TMP\2')
+        elsif client_path == '/gdc/projects/VCRFakeId'
```

Comment:
I don't like these elseifs since they are rather totally different rule than the one covered by `if client_path.match(uploads_regex) ...`. However not beeing expert on ruby way, still please consider how to make it less confusing.